__tests__/find-index.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable indent */
+const { findIndex } = require('../find-index');
+
+
+describe('findIndex', () => {
```

Comment:
Based on these tests, it looks like you're not quite hitting the goal of the findIndex function, which is, per the lab: `returns the index of the first item whose callback returns true or a truthy value` - so in these cases I'd expect the returned values to be 2 and 4, respectively. I'll probably have a couple more notes when I get to the function itself, but thought this would be a good spot to clarify.